kafka-ui-react-app/package.json

```diff
@@ -47,9 +47,9 @@
     ]
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "export REACT_APP_COMMIT=$(git rev-parse --short HEAD) && export REACT_APP_BUILD_DATE=$(date '+%F %H:%M:%S') && react-scripts start",
```

Comment:
I'm not sure it the correct place for such logic

Reply:
This is the most common way to do this task. There is another option to transfer this logic to webpack. But if we use vite now, I can investigate the question of how to move it there. Should I do this?

Reply:
How did it work before?

Reply:
We got some information from the request to backend:
https://api.github.com/repos/provectus/kafka-ui/releases/latest

But now there is no information we need there

Reply:
> How did it work before?

before that there was no build time, and we have to either parse it on the frontend part or to pass it via npm build, like we already do with the tag

Reply:
https://github.com/provectus/kafka-ui/blob/1b71ccb9755041e18ff540d5c57318243b38876d/kafka-ui-api/pom.xml#L404-L405

Reply:
I think all this info should be provided on build step
https://github.com/git-commit-id/git-commit-id-maven-plugin/blob/master/docs/using-the-plugin-in-more-depth.md